var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -111,7 +110,7 @@ def cmake_args(self):
             'compiler-rt'
         ]
         args = []
-        if self.spec.satisfies('@4.3.0:4.5.0'):
```

Comment:
@haampie , i think we do not the upper bound here... i added for 4.5.0 ... i think if self.spec.satisfies('@4.3.0:'):  will do

Reply:
Changed to  self.spec.satisfies('@4.3.0:'):